### ISSUE TITLE
Configure FTP links in Firefox to automatically use FireFTP

### ISSUE THREAD

**Issue by peter-rindfuss** [mimecuvalo/fireftp#186]
In 2.0.29, the option "Configure FTP links in Firefox to automatically use FireFTP" has vanished, and FireFTP is always used for FTP. This is inconvenient in many cases. Peter

**Comment by mimecuvalo** [comment]
Sorry, it's now turned on permanently because of new changes in Firefox that make it difficult to do otherwise.  In the future, there might be a way around it - patches happily welcome to fix this!

**Comment by Endor8** [comment]
Yes I miss this option too. I hope it will come back.
This is inconvenient in many cases.
For me it is an absolutely crucial option. I need it, so please add it.
Endor

**Comment by Ryushin** [comment]
Is there a way to do the opposite and make sure firefox never uses FireFTP for ftp links.  This is so annoying that I removed FireFTP.

**Comment by rv32** [comment]
Is there a way to get the previous version instead of this one?

**Comment by Endor8** [comment]
Yes see here: https://addons.mozilla.org/de/firefox/addon/fireftp/versions/

**Comment by ERDev** [comment]
Hi Mike,
No fix yet for this annoying feature?

**Comment by mimecuvalo** [comment]
Development on FireFTP is slowing down since Firefox is dropping support for it in version 57 later in the year.  Hence, motivation for working on it isn't the highest currently. I'll probably be porting FireFTP at least to PaleMoon so that the extension survives.
